docker-compose.yml

```diff
@@ -1,15 +1,19 @@
 mongodb:
-    image: mongo:3.1
+    image: mongo:3.4
     ports:
         - "27017"
     volumes:
         - "/data/db"
-
 girder:
-    build: .
+    build: ./girder
     dockerfile: Dockerfile
-    ports:
-        - "8080:8080"
     links:
         - "mongodb:mongodb"
-    command: -d mongodb://mongodb:27017/girder
\ No newline at end of file
+    command: -d mongodb://mongodb:27017/girder
+web:
+    build: ./nginx
+    dockerfile: Dockerfile
+    ports: 
+        - "8080:80"
+    links:
+        - "girder:girder"
```

Comment:
@zachmullen quick question on links: I created service:alias as girder:girder but I believe it should be girder:localhost and then use proxy_pass http://localhost:9000/ instead of proxy_pass http://girder:9000. 

The reason I am asking this that the current proxy_pass did not work for @matthewma7 but worked me. I have basic understanding of links and how it works but not in detail.